actors/market/src/lib.rs

```diff
@@ -1293,14 +1312,16 @@ fn deal_proposal_is_internally_valid(
     // Generate unsigned bytes
     let proposal_bytes = serialize(&proposal.proposal, "deal proposal")?;
 
-    extract_send_result(rt.send_simple(
+    extract_send_result(rt.send(
         &proposal.proposal.client,
         ext::account::AUTHENTICATE_MESSAGE_METHOD,
         IpldBlock::serialize_cbor(&ext::account::AuthenticateMessageParams {
             signature: signature_bytes,
             message: proposal_bytes.to_vec(),
         })?,
         TokenAmount::zero(),
+        None,
+        SendFlags::READ_ONLY,
```

Comment:
While we're touching this code, the method has a terribly misleading name. Rename it to `authenticate_proposal` or similar?